### PR TITLE
Add Mev Blocker kickbacks alert

### PR DESCRIPTION
**Pull request by harisang**
This PR adds a test that monitors for MEV Blocker kickbacks, and generates an alert if a large enough kickback is received. This is a follow-up PR on PR #79

### CODE REVIEW

File: src/constants.py [hunk 1]
```diff
@@ -25,8 +25,14 @@
 # cap parameter, per CIP-20, measured in ETH
 CAP_PARAMETER = 0.01
 
-# requests
+# threshold parameter to generate an alert when receiving kickbacks
```
Comment (fhenneke) [inline src/constants.py:28]: Should we move some of the more test specific constants into the test itself?

File: src/monitoring_tests/mev_blocker_kickbacks_test.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        block_number = self.web3_api.get_tx_block_number(tx_hash)
+        if block_number is None:
+            return False
+
+        total_eth_kickbacks = self.web3_api.get_eth_transfers_by_block_range(
+            block_number, block_number, MEV_BLOCKER_KICKBACKS_ADDRESS
+        )
+        if total_eth_kickbacks is None:
+            return False
+        log_msg = f'"Kickback of {total_eth_kickbacks} ETH received due to {tx_hash}"'
```
Comment (fhenneke) [inline src/monitoring_tests/mev_blocker_kickbacks_test.py:39]: This does not yet conform to how other tests report things. For example we could do:

```suggestion
        log_output = "\t".join(
            [
                "MEV Blocker kickbacks test:",
                f"Tx Hash: {tx_hash}",
                f"Kickback: {total_eth_kickbacks:.5f}ETH",
            ]
        )
```

**Review by fhenneke — CHANGES_REQUESTED** [review]
Not sure how to run that locally. In the repo on my computer this does not include the latest changes of main and thus does not have access to the new `get_eth_transfers_by_block_range` functionality. I could rebase locally but I wonder how you (or github via pylint) tested this branch.

**Comment by harisang** [comment]
> Not sure how to run that locally. In the repo on my computer this does not include the latest changes of main and thus does not have access to the new `get_eth_transfers_by_block_range` functionality. I could rebase locally but I wonder how you (or github via pylint) tested this branch.

Could you try again? Just did a rebase.

File: src/constants.py (hunk 1 above)
Comment (harisang) [reply src/constants.py:28]: Don't have a strong opinion on that. I could do that if you think the constants file has grown a lot.

Comment (fhenneke) [reply src/constants.py:28]: Then lets keep as is and refactor later if necessary.

**Review by fhenneke — APPROVED** [review]
I tested the code for hash `0xa8c5fae2995d74b2a5738b588aa758831b818f21bb283124a8c8c80f813bedec` and the results coincide with what I get in this [dune query](https://dune.com/queries/3166955/5285928?end_block_t6c1ea=18369463&start_block_t6c1ea=18369463).

LGTM.

**Pull request merged**